src/components/trace/hat-script-graph.ts

```diff
@@ -69,72 +77,30 @@ class HatScriptGraph extends LitElement {
 
   private render_trigger(config: Trigger, i: number) {
     const path = `trigger/${i}`;
-    const tracked = this.trace && path in this.trace.trace;
+    const track = this.trace && path in this.trace.trace;
     this.renderedNodes[path] = { config, path };
-    if (tracked) {
+    if (track) {
       this.trackedNodes[path] = this.renderedNodes[path];
     }
     return html`
       <hat-graph-node
         graphStart
+        ?track=${track}
         @focus=${this.selectNode(config, path)}
-        class=${classMap({
-          track: tracked,
-          active: this.selected === path,
-        })}
+        ?active=${this.selected === path}
         .iconPath=${mdiAsterisk}
-        tabindex=${tracked ? "0" : "-1"}
+        tabindex=${track ? "0" : "-1"}
```

Comment:
Maybe tabindex should actually be removed from the graphs entirely. I added it for accessibility, but in the current implementation it makes no sense and may actually be a hindrance.

I'll raise this as a wider issue somewhere.

Reply:
Yeah, that should probably be a seperate PR